code.js

```diff
@@ -1,15 +1,13 @@
 function binarySearch(list, element) {
-    list.sort(function(a, b){   // Check if the array is numerically sorted or not
-        return a-b;
-    });
+    list.sort((a,b) => a-b);
```

Comment:
I don't think that this is what the user would expect, and as I've said before, it makes everything much more expensive.